packages/by-name/kata/kata-runtime/0014-tools-don-t-clean-build-root-when-generating-rootfs.patch

```diff
@@ -0,0 +1,39 @@
+From 559b1962abae0eec37be0d2e788ad0d5fe5fecf4 Mon Sep 17 00:00:00 2001
+From: Markus Rudy <mr@edgeless.systems>
+Date: Tue, 1 Oct 2024 10:39:36 +0200
+Subject: [PATCH] tools: don't clean build root when generating rootfs
+
+---
+ tools/osbuilder/rootfs-builder/rootfs.sh | 9 +++------
+ 1 file changed, 3 insertions(+), 6 deletions(-)
+
+diff --git a/tools/osbuilder/rootfs-builder/rootfs.sh b/tools/osbuilder/rootfs-builder/rootfs.sh
+index 5fd77290d..b617ee3c6 100755
+--- a/tools/osbuilder/rootfs-builder/rootfs.sh
++++ b/tools/osbuilder/rootfs-builder/rootfs.sh
+@@ -398,12 +398,6 @@ build_rootfs_distro()
+ 		trap error_handler ERR
+ 	fi
+ 
+-	if [ -d "${ROOTFS_DIR}" ] && [ "${ROOTFS_DIR}" != "/" ]; then
+-		rm -rf "${ROOTFS_DIR}"/*
+-	else
+-		mkdir -p ${ROOTFS_DIR}
+-	fi
+-
+ 	if [ "${SELINUX}" == "yes" ]; then
+ 		if [ "${AGENT_INIT}" == "yes" ]; then
+ 			die "Guest SELinux with the agent init is not supported yet"
+@@ -719,9 +713,12 @@ EOF
+ 			[ -e ${fstab_file} ] && sed -i '/\/run/d' ${fstab_file}
+ 			echo "tmpfs /run tmpfs nodev,nosuid,size=50% 0 0" >> ${fstab_file}
+ 
++            info "Enabling systemd-remount-fs.service"
+ 			kata_systemd_target="${ROOTFS_DIR}/usr/lib/systemd/system/kata-containers.target"
++            ls -l ${kata_systemd_target}
+ 			grep -qE "^Requires=.*systemd-remount-fs.service.*" ${kata_systemd_target} || \
+ 				echo "Requires=systemd-remount-fs.service" >> ${kata_systemd_target}
++            info "Done enabling systemd-remount-fs.service"
```

Comment:
Were these debug statements checked in intentionally?

Reply:
Nope, didn't mean to.